presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreenState.kt

```diff
@@ -0,0 +1,9 @@
+package me.androidbox.presentation.event.screen
+
+import android.net.Uri
+import androidx.compose.runtime.mutableStateListOf
+import androidx.compose.runtime.snapshots.SnapshotStateList
+
+data class EventScreenState(
+    val listOfPhotoUri: SnapshotStateList<Uri> = mutableStateListOf<Uri>()
```

Comment:
Just curious

Why are you using SnapshotStateList<Uri> instead of List<Uri> ?

Reply:
@MKiperszmid  I did try and use a List but cannot add items to the List. Any ideas?